.github/workflows/docker-image-publish.yml

```diff
@@ -20,42 +20,35 @@ jobs:
             - name: Update git sha
               run: echo "GIT_SHA = '${GITHUB_SHA}'" >posthog/gitsha.py
 
-            - name: Set up QEMU
-              if: github.repository == 'PostHog/posthog'
-              uses: docker/setup-qemu-action@v1
-
-            - name: Set up Docker Buildx
-              if: github.repository == 'PostHog/posthog'
-              uses: docker/setup-buildx-action@v1
+            - name: Set up Depot CLI
+              uses: depot/setup-action@v1
 
             - name: Login to DockerHub
-              if: github.repository == 'PostHog/posthog'
```

Comment:
The whole workflow is already guarded with `if: github.repository == 'PostHog/posthog'`, so I've removed the `if`s from individual steps.